isis/appdata/import/RosOsiris.tpl

```diff
@@ -0,0 +1,272 @@
+      {% set filterTwoWidth = 84.5 %}
+    {% else if filterName.1 == "Blue" %}
+      {% set filterTwoCenter = 480.7 %}
+      {% set filterTwoWidth = 74.9 %}
+    {% else if filterName.1 == "Red" %}
+      {% if InstrumentId == "OSIWAC" %}
+       {% set filterTwoCenter = 629.8 %}
+        {% set filterTwoWidth = 156.8 %}
+      {% else if InstrumentId == "OSINAC"%}
+        {% set filterTwoCenter = 743.7 %}
+        {% set filterTwoWidth = 64.1 %}
+      {% endif %}
+    {% else if filterName.1 == "Green" %}
+      {% if InstrumentId == "OSIWAC" %}
+        {% set filterTwoCenter = 537.2 %}
+        {% set filterTwoWidth = 63.2 %}
+      {% else if InstrumentId == "OSINAC"%}
+        {% set filterTwoCenter = 537.7 %}
+        {% set filterTwoWidth = 62.4 %}
+      {% endif %}
+    {%else if filterName.1 == "Empty" %}
+        {% set filterTwoCenter = 0.0 %}
+        {% set filterTwoWidth = 0.0 %}
+    
+
+    {% else if filterName.1 == "UV245" %}
+      {% set filterTwoCenter = 246.2 %}
+      {% set filterTwoWidth = 14.1 %}
+    {% else if filterName.1 == "CS" %}
+      {% set filterTwoCenter = 259.0%}
+      {% set filterTwoWidth = 5.6%}
+    {% else if filterName.1 == "UV295" %}
+      {% set filterTwoCenter = 295.9 %}
+      {% set filterTwoWidth = 10.9 %}
+    {% else if filterName.1 == "OH-WAC" %}
+      {% set filterTwoCenter = 309.7 %}
+      {% set filterTwoWidth = 4.1 %}
+    {% else if filterName.1 == "UV325" %}
+      {% set filterTwoCenter = 325.8 %}
+      {% set filterTwoWidth = 10.7 %}
+    {% else if filterName.1 == "NH" %}
+      {% set filterTwoCenter = 335.9 %}
+      {% set filterTwoWidth = 4.1 %}
+    {% else if filterName.1 == "UV375" %}
+      {% set filterTwoCenter = 375.6 %}
+      {% set filterTwoWidth = 9.8 %}
+    {% else if filterName.1 == "CN" %}
+      {% set filterTwoCenter = 388.4 %}
+      {% set filterTwoWidth = 5.2 %}
+    {% else if filterName.1 == "NH2" %}
+      {% set filterTwoCenter = 572.1 %}
+      {% set filterTwoWidth = 11.5 %}
+    {% else if filterName.1 == "Na" %}
+      {% set filterTwoCenter = 590.7 %}
+      {% set filterTwoWidth = 4.7 %}
+    {% else if filterName.1 == "VIS610" %}
+      {% set filterTwoCenter = 612.6 %}
+      {% set filterTwoWidth = 9.8 %}
+    {% else if filterName.1 == "OI" %}
+      {% set filterTwoCenter = 631.6 %}
+      {% set filterTwoWidth = 4.0 %}
+
+
+    {% endif %}
+
+    FilterNumber       = {{SR_MECHANISM_STATUS.FILTER_NUMBER.Value}}
+    CombinedFilterName = {{SR_MECHANISM_STATUS.FILTER_NAME.Value}}
+    FilterId           = {{SR_MECHANISM_STATUS.FILTER_NUMBER.Value}}
+    FilterOneName      = {{filterOneName}}
+    FilterOneCenter    = {{filterOneCenter}} <nanometers>
+    FilterOneWidth     = {{filterOneWidth}} <nanometers>
+    FilterTwoName      = {{filterTwoName}}
+    FilterTwoCenter    = {{filterTwoCenter}} <nanometers>
+    FilterTwoWidth     = {{filterTwoWidth}} <nanometers>
+  End_Group
+
+  Group = Kernels
+    NaifFrameCode = -226111
```

Comment:
Need a check for if the instrument id if its WAC or NAC and change the NaifFrameCode accordingly. See these lines in rososiris2isis https://github.com/USGS-Astrogeology/ISIS3/blob/6be37123d506479fcdd7660630d2048f63a72cb2/isis/src/rosetta/apps/rososiris2isis/main.cpp#L151-L156